app/src/main/java/com/m2049r/xmrwallet/SendFragment.java

```diff
@@ -352,7 +352,7 @@ private void prepareSend() {
         int mixin = Mixins[sMixin.getSelectedItemPosition()];
         int priorityIndex = sPriority.getSelectedItemPosition();
         PendingTransaction.Priority priority = Priorities[priorityIndex];
-        Log.d(TAG, dst_addr + "/" + paymentId + "/" + amount + "/" + mixin + "/" + priority.toString());
+        Timber.d(dst_addr + "/" + paymentId + "/" + amount + "/" + mixin + "/" + priority.toString());
```

Comment:
Why not
```Java
Timber.d("%s/%s/%s/%s/%s", dst_addr, paymentId, amount, mixin, priority.toString());
```
?
In general many instances use %s and many others (still) use +. We should try to be consistent, sooner or later,

Reply:
Reason is that Timber provides a Lint rule for `"bla " + value` but not for `value + " " + value + " "` but you are right it should be consistent and I will change it.